src/components/bubble/bubble.spec.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Bubble from './Bubble';
+import {shallow} from 'enzyme';
+
+
+test('render', () => {
+  const bubble = shallow(
+    <Bubble>Some text</Bubble>
+  );
+
+  expect(bubble.hasClass('sg-bubble')).toEqual(true);
+});
+
+test('render top', () => {
+  const bubble = shallow(
+    <Bubble direction='top'>Some text</Bubble>
+  );
+
+  expect(bubble.hasClass('sg-bubble--top')).toEqual(true);
+});
+
+
+test('render top start', () => {
+  const bubble = shallow(
```

Comment:
**shallow** is render type from enzyme. It's render component without inner components and without mounting states. It's fastest render method in test suite.
http://airbnb.io/enzyme/